roles/config-quay-enterprise/tasks/configure_systemd.yml

```diff
@@ -17,5 +17,5 @@
   notify: "Restart quay service"
```

Comment:
May want to do another set of notifies here for the various services that you are potentially changing. I.e.: the services - for example DB - is a dependency for `quay` and will be started if it isn't, but in the event that the service is already running and this task make a change to the config/environment for that service it won't know to reload/restart the service.